### PR TITLE
feat: add MESSAGE ControllerType

### PR DESCRIPTION
**Pull request by fengmk2**

### CODE REVIEW

File: core/controller-decorator/src/model/types.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ export enum ControllerType {
   HTTP = 'HTTP',
   SOFA_RPC = 'SOFA_RPC',
   MGW_RPC = 'MGW_RPC',
+  MESSAGE = 'MESSAGE',
+  SCHEDULE = 'SCHEDULE',
```
Comment (killagu) [inline core/controller-decorator/src/model/types.ts:12]: https://github.com/eggjs/tegg/blob/master/core/schedule-decorator/src/decorator/Schedule.ts 

schedule 已经有了。

Comment (fengmk2) [reply core/controller-decorator/src/model/types.ts:12]: SCHEDULE 没有走 Controller 模式？

File: core/controller-decorator/src/model/types.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ export enum ControllerType {
   HTTP = 'HTTP',
   SOFA_RPC = 'SOFA_RPC',
   MGW_RPC = 'MGW_RPC',
+  MESSAGE = 'MESSAGE',
```
Comment (fengmk2) [inline core/controller-decorator/src/model/types.ts:11]: @killagu 先只加 MESSAGE

**Review by killagu — APPROVED** [review]
LGTM

**Comment by fengmk2** [comment]
Successfully published:
 - @eggjs/aop-decorator@3.15.0
 - @eggjs/controller-decorator@3.15.0
 - @eggjs/core-decorator@3.15.0
 - @eggjs/egg-module-common@3.15.0
 - @eggjs/eventbus-decorator@3.15.0
 - @eggjs/standalone-decorator@3.15.0
 - @eggjs/tegg@3.15.0
 - @eggjs/tegg-aop-plugin@3.15.0
 - @eggjs/tegg-aop-runtime@3.15.0
 - @eggjs/tegg-background-task@3.15.0
 - @eggjs/tegg-common-util@3.15.0
 - @eggjs/tegg-config@3.15.0
 - @eggjs/tegg-controller-plugin@3.15.0
 - @eggjs/tegg-dynamic-inject@3.15.0
 - @eggjs/tegg-dynamic-inject-runtime@3.15.0
 - @eggjs/tegg-eventbus-plugin@3.15.0
 - @eggjs/tegg-eventbus-runtime@3.15.0
 - @eggjs/tegg-lifecycle@3.15.0
 - @eggjs/tegg-loader@3.15.0
 - @eggjs/tegg-metadata@3.15.0
 - @eggjs/tegg-orm-decorator@3.15.0
 - @eggjs/tegg-orm-plugin@3.15.0
 - @eggjs/tegg-plugin@3.15.0
 - @eggjs/tegg-runtime@3.15.0
 - @eggjs/tegg-schedule-decorator@3.15.0
 - @eggjs/tegg-schedule-plugin@3.15.0
 - @eggjs/tegg-standalone@3.15.0
 - @eggjs/tegg-transaction-decorator@3.15.0
lerna success published 28 packages

**Pull request merged**